nomic-bot/command-processors/join-processor.js

```diff
@@ -0,0 +1,38 @@
+(function () {
+    'use strict';
+    var _ = require('lodash'),
+        stringFormat = require('string-format'),
+        github = require('../utils/github.js'),
+        voteProcessor = require('../command-processors/vote-processor.js'),
+        rollProcessor = require('../command-processors/roll-processor.js'),
+        joinProcessor = {
+            messages: {
+                notOwner: '@{login}, you cannot join in an issue you did not create.',
+                playerCreated: '{login} player has been created!'
+            },
+            initialPoints: '/roll 1d12 + 12',
+            processJoin: function (commentsUrl, userLogin, requestBody) {
+                if (requestBody.issue.user.login !== userLogin) {
+                    return github.sendCommentMessage(commentsUrl, stringFormat(joinProcessor.messages.notOwner, {login: userLogin}));
+                }
+                
+                return github.getPlayerData()
+                .then(function (playerData) {
+                    var activePlayers = playerData.activePlayers;
+                    
+                    var player = {
+                        name: userLogin,
+                        points: rollProcessor.sum(joinProcessor.initialPoints, {})
+                    };
+
+                    activePlayers.push(player);
```

Comment:
I spoke with @allensb offline about this. We want a check for if the player exists before adding them just so people don't accidentally add themselves twice.